README.md

```diff
@@ -8,5 +8,70 @@ Git: Kamon
 Client: Kamon, Mandy, Jana
 Server: Erwin, Sach, Siobhan
 Full-stack: Justin, Prem
+<enter>
 
+## Build and run client
 
+<pre>
+<code>
+npm run build
+npm start
+</code>
+</pre>
+
+
+
+# Installation - Client Side
+
+Open the terminal and enter your 'stupid-questions-client' repository.
+
+Run 'npm i' to install appropriate packages in the package.json:
+
+```
+npm i
+```
+
+Once the packages have been installed, the webpack must be built using 'npm run build'
+
+```
+npm run build
+```
+
+Then open a new tab in the terminal and run the server
+
+```
+npm run server
+```
+
+
+##Creating seed dummy Data- Client side
+<ul>
```

Comment:
This section doesn't format well. Try using blocks for code delimited with three backticks. For bulleted lists, you can just use the minus sign:
```
- like this
- and this
```